INFO:root:gpu device = cuda:0
INFO:root:epoch 0 lr 4.442566e-02
Traceback (most recent call last):
  File "main.py", line 295, in <module>
    save_model_str=get('run_dir'))
  File "main.py", line 237, in create_run_ensemble
    train_acc, train_obj = train(train_loader, ensemble_model, criterion, optimizer, grad_clip=ensemble_config['grad_clip_value'])
  File "/home/rkohli/aml-project/src/train.py", line 128, in train
    logits = model(input)
  File "/home/rkohli/anaconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/rkohli/aml-project/src/model.py", line 297, in forward
    logging.info("Output size: {}".format(out.size()))
NameError: name 'logging' is not defined
